### ISSUE TITLE
bitcoins send but never received in vendors wallet

### ISSUE THREAD

**Issue by fridolink89** [bitpay/copay#9106]
Hallo,

I have a problem. I send btc to a Wallet Adress of a vendor. But the money was never received. 

Here is a copy from insight bitpay.

Please can anynone help. I don`t know what is going wrong.

Thank you for every help.

 Address 1666dvWhb2952texdL8i11Z8WAw8Eut2px

Summary confirmed
Total Received | 0.009 BTC
-- | --
Total Sent | 0.009 BTC
Final Balance | 0 BTC
No. Transactions | 2

Sync Progress | 100%                     Complete
-- | --
Current Sync Status | finished
Start Date | a few seconds ago
Initial Block Chain Height | 540832
Synced Blocks |  
Skipped Blocks (previously synced) |  
Sync Type | bitcore node

Last Block Hash (Bitcoind) | 0000000000000000001ba37aab00b8ada4e73bf5ee879d0dd35b910033dd8d67
-- | --
Current Blockchain Tip (insight) | 0000000000000000001ba37aab00b8ada4e73bf5ee879d

**Comment by BGalliotti** [comment]
Hello @fridolink89 
Thanks for reaching out to us.

- **Please send us the BitPay/Copay Wallet ID and the transaction hash / ID or the link so we can further investigate and address your inquiry.**

- To get your Wallet ID, open your Copay/BitPay application, then go to `Settings` > [Your wallet] > `More Options` > `Wallet Information`. 
On your phone, if you tap on the wallet ID it will copy it to the clipboard so you can paste it in your e-mail. 
You can do the same on your computer by clicking on the wallet ID.

- The transaction hash / ID is a alphanumeric chain like this: **d0a23876de7cf87574b5cda7557bcc8af5d84f3fd1e6f4c6063fd6b72b0e69ae**

**We look forward to hearing from you.**

Please let us know if you have any additional questions.
Kind regards.

**Comment by fridolink89** [comment]
Wallet ID: 25a603f5-eacb-4767-818a-e88ada34790b

Transaction ID: 6eaea5e69458232436d69b1ce9c9f14fc714dbe5114bd2a1db09e0575e865593

hope that will help.

**Comment by BGalliotti** [comment]
Hello @fridolink89 
Thanks for reaching out to us.

We have checked your wallet and the transaction is showing up.

**Please open up your BitPay app and try the following steps:**

- Wallet > Preferences (top right corner icon) > Transaction History > Clear cache
- Wallet > Preferences (top right corner icon) > Wallet Addresses > Scan addresses for funds

Please let us know if you have any additional questions.
Kind regards.

**Comment by fridolink89** [comment]
Hello,

I have done this before. And now I did it again. But it don`t helps.

**Comment by BGalliotti** [comment]
Hello @fridolink89 
Thanks for reaching out to us.

Here is your wallet report showing the transaction **6eaea5e69458232436d69b1ce9c9f14fc714dbe5114bd2a1db09e0575e865593**:

```
* Wallet info
  ID: 25a603f5-eacb-4767-818a-e88ada34790b Coin: btc
  LIVENET 1-of-1 [BIP44 P2PKH] wallet (status: complete)
  Created on: Thu Sep 06 2018 16:59:53 GMT-0300

* Balance
  Total: 0.00000000 btc (0.00000000 btc locked)
  Confirmed: 0.00000000 btc (0.00000000 btc locked)
  Available: 0.00000000 btc (0.00000000 btc confirmed / 0.00000000 btc unconfirmed)

* No pending tx proposals.

* Tx history
  [Mon Sep 17 2018 17:16:00 GMT-0300] c0ae6aab97cda385d507e2a3ee17e185604219f2e84e91df1524b577587b26cd => sent 0.01025771 btc (fee/kb 0.00026836 btc) (543 confirmations)
  [Mon Sep 17 2018 07:27:32 GMT-0300] 1f94f4e317472cc5b8f41bc7260ce4553fe5539703818486cd2d948a8fe6fedd <= received 0.01025771 btc (fee/kb 0.00004449 btc) (600 confirmations)
  [Tue Sep 11 2018 15:45:21 GMT-0300] 6cbe34c5a5a01fdc9b2e397bf03a9483e501c73df717816bbed18cfc74bbc273 => sent 0.01415268 btc (fee/kb 0.00098354 btc) (1409 confirmations)
  [Tue Sep 11 2018 15:26:52 GMT-0300] a7f9d13bfe43b18fc0a82ef0c77ca7ac570d0cea9c127d4b28bf443121bfd1c0 <= received 0.01415268 btc (fee/kb 0.00021572 btc) (1410 confirmations)
  [Fri Sep 07 2018 23:01:30 GMT-0300] 84ed9fef00344e57e4757c288cfbb01ce0b17ffc999369beed2c00f6229c4cd6 => sent 0.00808479 btc (fee/kb 0.00098505 btc) (1958 confirmations)
  [Thu Sep 06 2018 19:30:02 GMT-0300] 6eaea5e69458232436d69b1ce9c9f14fc714dbe5114bd2a1db09e0575e865593 => sent 0.00900000 
btc (fee/kb 0.00028738 btc) (2129 confirmations)
  [Thu Sep 06 2018 18:13:58 GMT-0300] 8c386a7360e50f0d323ed2e867e509dff422d7dd2aa3909e3cacfc1f3c607e60 <= received 0.01714945 btc (fee/kb 0.00024717 btc) (2138 confirmations)
```
As you can see, the transaction is there.

Furthermore, as you can see on the transaction [here](https://live.blockcypher.com/btc/tx/6eaea5e69458232436d69b1ce9c9f14fc714dbe5114bd2a1db09e0575e865593/), the funds are already spent (regardless of who owned them). 

Besides, you have no funds, in case that was your concern. Regarding your balance, there is nothing to fix, everything is fine.

The solution we previously provided is for refreshing the cache and force the wallet to re-sync in case the transactions are not properly showing up and will not solve anything related to the balance (other than showing it correctly).

Please let us know if you have any additional questions.
Kind regards.
